bin/ffscreencast

```diff
@@ -1053,7 +1093,7 @@ if [ "${RECORD_S}" = "yes" ]; then
 	if [ ${#CHOSEN_S_NUM} -gt 0 ]; then
 		if ! screen_device_exists "$CHOSEN_S_NUM"; then
 			echo "Screen recording device: '${CHOSEN_S_NUM}' does not exist."
-			exit -1
+			exit 1
```

Comment:
@Dark-Kernel I'm curious about this change? Again, it's a slight change, but it's not commented in the PR so I'm uncertain as to the rationale.

```sh
$ bash -c 'exit -1'
$ echo $?
255

$ bash -c 'exit 1'
$ echo $?
1
```

Reply:
what difference it makes?

Reply:
Again, this is just me noticing an unacknowledged change in default behavior.

My interest in this project is to be able to easily record my screen on Linux. I use `i3wm`, which is a very customizable window manager.

Right now it is nice that `ffscreencast` has three different exit values:

* `0` -- no errors; my screen recording is where I want it to be
* `1` -- some sort of error occurred, probably while running
* `255` -- the script couldn't even start doing its work

Changing from `exit -1` to `exit 1` here means I get a little less information from the exit value of `ffscreencast`.

Reply:
Cool! Learned something new. Will change it.